internal/middleware/middleware_test.go

```diff
@@ -155,7 +159,7 @@ func TestValidateRedirectURI(t *testing.T) {
 			handler.ServeHTTP(rr, req)
 			if rr.Code != tt.status {
 				t.Errorf("Status code differs. got %d want %d", rr.Code, tt.status)
-				t.Errorf("%s", rr.Body)
+				// t.Errorf("%s", rr.Body)
```

Comment:
@desimone is this commenting on purpose?

Reply:
oops!